### ISSUE TITLE
Cannot escape backtick(`) in fenced code block

### ISSUE THREAD

**Issue by lzh9102** [gnab/remark#156]
I am tryping to show a code example in my slide using github-style fenced blocks:

     ```bash
     echo "message" > log-`date --iso-8601`.txt
     ```
I am aware that the backtick(`) is used to highlight part of the code example in Remark. I tried to escape the backticks like this:

     ```bash
     echo "message" > log-\`date --iso-8601\`.txt
     ```

But the code block is rendered like this:

```bash
echo "message" > log-\`date --iso-8601\`.txt
```
Is there a way to write the code example so that it will be rendered as

```bash
echo "message" > log-`date --iso-8601`.txt
```

Thank you.

**Comment by leonardinius** [comment]
It's also hit me as well.

Any way to escape those `\``?

**Comment by VadimPushtaev** [comment]
You can use this as a temporary workaround: https://gist.github.com/VadimPushtaev/2e123b16798a0a815caf

**Comment by leonardinius** [comment]
@VadimPushtaev thanks, it was helpful.

**Comment by gnab** [comment]
#220 fixed this and was included in version 0.11.0. You can now escape backtics with a backslash.
